src/dataframe/dataframe.jl

```diff
@@ -121,6 +121,8 @@ function DataFrame(pairs::Pair{Symbol,<:Any}...; makeunique::Bool=false)::DataFr
     DataFrame(columns, Index(colnames, makeunique=makeunique))
 end
 
+DataFrame(dict::AbstractDict) = convert(DataFrame, dict)
```

Comment:
You appear to have missed [my previous comment](https://github.com/JuliaData/DataFrames.jl/pull/1364#discussion_r171516203).

Reply:
? Wasn't it regarding splatting?  This is no longer splatting.  Can you explain what you'd like me to change here?

Reply:
I mean this:
> On 0.7, the recommendation added in JuliaLang/julia#23273 is to define constructors, and have convert methods call constructors explicitly (so the inverse of what we do now, mostly).

Reply:
Oh, sorry I had missed that.  Will change now.